src/runtime_src/core/edge/drm/zocl/zocl_xclbin.c

```diff
@@ -916,15 +916,14 @@ zocl_xclbin_load_pskernel(struct drm_zocl_dev *zdev, void *data, uint32_t slot_i
 
         mutex_lock(&slot->slot_xclbin_lock);
 	/* Check unique ID. Avoid duplicate PL xclbin */
-	if ((slot->slot_type == ZOCL_SLOT_TYPE_PHY) &&
-	    (zocl_xclbin_same_uuid(slot, &axlf_head->m_header.uuid))) {
+	if (zocl_xclbin_same_uuid(slot, &axlf_head->m_header.uuid)) {
```

Comment:
This checks will block loading the same PS xclbin also. 
This  will not work for BUILT-IN PS kernel

Reply:
As per offline discussion. Please ignore the above changes,